Frontend/src/pages/Dashboard.jsx

```
@@ -1,35 +1,14 @@
-        <button type="button" onClick={logout} className="bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 rounded-lg shadow-md transition duration-300">
-          Logout
-        </button>
-      </div>
-
-      {/* Suggested Users Section */}
-      <div className="p-6">
-        <SuggestedUsers />
-      </div>
+    <div>
+      <h2>Welcome, {user.email}</h2>
+      <button onClick={logout}>Logout</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)